docs/src/getting-started/how-it-compares.md

```diff
@@ -1,42 +1,43 @@
 # How it Compares
 
-Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between it and the Fuel indexer. Generally, the biggest conceptual differences between the two are as follows:
+- Since many users may be familiar with indexing by using a solution like The Graph, it may be helpful to provide a comparison between The Graph and the Fuel indexer.
 
-- The Fuel indexer does not support publishing indexers to decentralized networks. That being said, the Fuel indexer is developed with the intention of allowing anyone to run an instance and host whatever indexers they want.
-- Contracts on the Fuel network can be indexed without the need to add additional events. Whether a user wanted to index information from the very start of a contract or a recent block, the process is the same: define the schema and manifest, write custom handler logic, and deploy to an indexer service.
+## Differential Value
 
-Legend:
+- Generally, the biggest conceptual differences between other indexer services such as The Graph, and Fuel's indexer service include: indexing speed, general ease of use, and the type of data that can be indexed.
+
+### Speed of indexing
+
+- Using Fuel's indexers, users can index about 30 blocks per second on a standard Macbook Pro on an M1 chip. This type of indexing speed is a boon to smart contract authors who need to iterate quickly while building dApps.
+
+### General usability
 
-- 🟩 : Functionally complete
-- 🟨 : Partially complete
-- 🟥 : Planned but incomplete
-- ⛔ : Not planned
+- Unlike other indexing services, with the `forc index` CLI tool, users can create, deploy, update, re-deploy, remove, and check the status of their indexers. The ability to completely manage, maintain, and improve remote indexers without having to open any files, or edit any source code completely sets Fuel's indexer apart from other services in the space.
+
+### What you can index
+
+- The Fuel indexer is tailored for compatibility with the FuelVM. Meaning, whereas with services like The Graph, indexer operators are limited to some of the primitives of the EVM - with the Fuel indexer, users get a much richer set of indexable abstractions provided by the FuelVM (e.g., predicates, transaction receipts, etc).
```

Comment:
Consider the following:
> The Fuel indexer is tailored for compatibility with the FuelVM. This means that instead of being limited to the primitives of the Etherem virtual machine (EVM), users of the Fuel indexer gain access to a much richer set of indexable abstractions provided by the FuelVM (e.g. predicates, transaction receipts, etc).